src/Sentry/Protocol/Envelopes/Envelope.cs

```diff
@@ -17,10 +17,6 @@ namespace Sentry.Protocol.Envelopes
     /// </summary>
     public sealed class Envelope : ISerializable, IDisposable
     {
-        private const string SdkKey = "sdk";
-        private const string EventIdKey = "event_id";
-        private const string SentAtKey = "sent_at";
```

Comment:
Just out of curiosity, why not a `const` anymore?

Reply:
This was part of a refactoring not specifically to dynamic sampling, but came up when adding the `"trace"` key.  I noted several existing inconsistencies, such as not having constants for `"name"` and `"version"` keys.  The decision was either add them for all, or none.  In general, we've decided to inline most key names in other parts of the SDK, so made sense to do it here as well.